test/content/support.js

```diff
@@ -202,39 +202,18 @@ function resetDB() {
  * Equivalent to JSON.stringify, except that object properties are stringified
  * in a sorted order.
  */
-function stableStringify(obj, level, label) {
-	if (!level) level = 0;
-	let indent = '\t'.repeat(level);
-	
-	if (label) label = JSON.stringify('' + label) + ': ';
-	else label = '';
-	
-	if (typeof obj == 'function' || obj === undefined) return null;
-	
-	if (typeof obj != 'object' || obj === null) return indent + label + JSON.stringify(obj);
-	
-	if (Array.isArray(obj)) {
-		let str = indent + label + '[';
-		for (let i=0; i<obj.length; i++) {
-			let json = stableStringify(obj[i], level + 1);
-			if (json === null) json = indent + '\tnull'; // function
-			str += '\n' + json + (i < obj.length-1 ? ',' : '');
+function stableStringify(obj) {
+	return JSON.stringify(obj, function(k, v){
+		if (v && "object" === typeof v && !Array.isArray(v)) {
```

Comment:
Looks good to squash, but before you do that, done formatting nits:
* `typeof v == "object"` (picking on the order, not so much the equals. It just looks odd that way, imo)
* space before `{` in the line above
* space after comma and semicolons in the for loop below

And also wait for @dstillman's ok, since he's the one to merge this.